airrohr-firmware/Readme.md

```diff
@@ -57,6 +57,7 @@ Installierbar über Arduino IDE (Menü Sketch -> Bibliothek einbinden -> Bibliot
 * [Adafruit BMP280 library](https://github.com/adafruit/Adafruit_BMP280_Library) (1.0.2) (BSD)
 * [Adafruit BME280 library](https://github.com/adafruit/Adafruit_BME280_Library) (1.0.5) (BSD)
 * [DallasTemperature](https://github.com/milesburton/Arduino-Temperature-Control-Library) (3.7.6)
+	* Dependency[OneWire] (https://github.com/PaulStoffregen/OneWire)
```

Comment:
Please remove the tab in front of the new dependency line, and provide a version of the OneWire library you are using

Reply:
Also, please add the dependency to platformio.ini. See the following link which seems to reference the library and version you are using:

http://platformio.org/lib/show/1/OneWire/installation

Reply:
I added this tab because DallasTemperature has dependency for oneWire.
(I will change this)